sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -20,10 +20,15 @@
 #include <string>
 #include <vector>
 
+// Declarations of a Windows PCCERT_CONTEXT structure to avoid including the Windows SDK headers.
+struct _CERT_CONTEXT;
+using PCCERT_CONTEXT = const _CERT_CONTEXT*;
```

Comment:
optional, in case you like it better
```suggestion
typedef struct _CERT_CONTEXT *const _CERT_CONTEXT;
```

Reply:
Sorry for the noise - that's not what it would do.

Reply:
Actually it's a good idea. Implemented.

Reply:
Nice! What you've done is better than what I've proposed.
I found no `typedef` syntax to alias a pointer to const using single statement. The one that I proposed would not declare a pointer to const.
The one that you did - does, and it is more readable, and more modern.